README.md

```diff
@@ -26,7 +27,8 @@ on:
   workflow_dispatch:
 
 env:
-  GITHUB_TOKEN: ${{ secret.PAT }}
+  GITHUB_PAT: ${{ secret.PAT }}
+  GITHUB_TOKEN: ${{ github.token }}
```

Comment:
PAT and TOKEN? Usually it's one or the other

Reply:
Not totally sure if it is a bug or an expected security behavior, I should open an issue in `cli/cli`, but:

- `gh api` for security events doesn't work with TOKEN - even if I set `permissions: write-all` in the flow https://github.com/pitch-io/clojure-dependabot/blob/38ee7cef0faf0c60c9e5df1c1b1182162f58d3e1/dependabot_alerts.sh#L5
- if I set only PAT, PRs will be created by the PAT's owner

So, if you want to have PRs opened by `github-actions (bot)`, and not the PAT's owner, and if you want to list the security events, you need both.

Reply:
I saw the doc update, thanks for addressing this!